Bytes to bytesNN conversion (#11047)
ethereum/solidity: docs/control-structures.rst

#. 0x12; If you divide or modulo by zero (e.g. ``5 / 0`` or ``23 % 0``).
#. 0x21: If you convert a value that is too big or negative into an enum type.
#. 0x22: If you access a storage byte array that is incorrectly encoded.

> #. 0x23: If you convert ``bytes`` array to fixed bytes type and size of fixed bytes is smaller than array length.

Reviewer: ```suggestion
#. 0x23: If you convert a ``bytes`` array to a ``bytesN`` type and the array is longer than ``N`` bytes.
```
